### PR TITLE
Makes it so changeling legs copy digitigrade/normal, try number 2

### PR DESCRIPTION
**Pull request by Seris02**
## About The Pull Request

Last attempt was #58852, failed because I had IRL things and stuff.

Makes it so changeling legs copy digitigrade/normal.

## Why It's Good For The Game

bug fixes

## Changelog
:cl:
fix: changeling legs now copy digitigrade/normal legs
/:cl:

### CODE REVIEW

File: code/modules/antagonists/changeling/changeling.dm [hunk 1]
```diff
@@ -516,6 +516,10 @@
 	user.socks = chosen_prof.socks
 
 	chosen_dna.transfer_identity(user, 1)
+	if (DIGITIGRADE in chosen_dna.species.species_traits)
+		user.Digitigrade_Leg_Swap(FALSE)
+	else
+		user.Digitigrade_Leg_Swap(TRUE)
```
Comment (Mothblocks) [inline code/modules/antagonists/changeling/changeling.dm:522]:
```suggestion
	user.Digitigrade_Leg_Swap(!(DIGITIGRADE in chosen_dna.species.species_traits))
```

**Comment by Greniza** [comment]
Make sure that swapping to or from digitigrade takes off someone's shoes.

File: code/modules/antagonists/changeling/changeling.dm (hunk 1 above)
Comment (Seris02) [reply code/modules/antagonists/changeling/changeling.dm:522]: ...what

this was what my last PR was, and you asked for the changes I made to this PR, why

**Comment by Seris02** [comment]
> Make sure that swapping to or from digitigrade takes off someone's shoes, so that if there's a change, someone isn't wearing digitigrade shoes when they shouldn't or vise versa

the digitigrade_leg_swap proc already does that

**Comment by Mothblocks** [comment]
Undraft if ready.

**Comment by Seris02** [comment]
Fairly certain I can't undraft it since you're the one who marked it as a draft, I got no option to undraft it.

**Comment by Mothblocks** [comment]
You don't see this?
image

**Comment by Seris02** [comment]
Ah, you’re right I’m just blind

**Pull request merged**